src/components/Button/SegmentedButton.md

```diff
@@ -1,20 +1,33 @@
+Use the segmented button to display a group of related controls in a single line. Combine them with the dropdown to create complex menu controls.
+
+View the [Rivet documentation for Segmented Buttons](https://rivet.uits.iu.edu/components/forms/buttons-segmented/).
+
 ### Default Style
 ```jsx
 <SegmentedButton>
-    <Button onClick={() => { console.log('Click!') }}>Primary One</Button>
-    <Button onClick={() => { console.log('Click!') }}>Primary Two</Button>
-    <Button onClick={() => { console.log('Click!') }}>Primary Three</Button>
+    <Button onClick={() => { console.log('Click 1!') }}>Primary One</Button>
+    <Button onClick={() => { console.log('Click 2!') }}>Primary Two</Button>
+    <Button onClick={() => { console.log('Click 3!') }}>Primary Three</Button>
+</SegmentedButton>
+```
+
+### Secondary Modifier
+```jsx
+<SegmentedButton>
+    <Button modifier="secondary" onClick={() => { console.log('Click 1!') }}>Secondary One</Button>
+    <Button modifier="secondary" onClick={() => { console.log('Click 2!') }}>Secondary Two</Button>
+    <Button modifier="secondary" onClick={() => { console.log('Click 3!') }}>Secondary Three</Button>
```

Comment:
This example is fine, but nearly every other example in the project shows how to emulate the examples from the Rivet core documentation so this one does not match up with those.